sourcetools/j9nls/com/ibm/oti/NLSTool/J9NLS.java

```diff
@@ -776,7 +776,7 @@ public void appendMsg(String moduleName, String key) {
 		public void writeHeader(String headerName, String moduleName) {
 			String header = headerName.replace('.', '_');
 			append("/*\n");
-			append(" * Copyright (c) 1991, 2017 IBM Corp. and others\n");
+			append(" * Copyright (c) 1991, 2019 IBM Corp. and others\n");
```

Comment:
Please revert this to 2017

Reply:
@fjeremic -- fwiw, I'm pretty sure your script would also trip on this...

Reply:
Yikes! Good catch!